src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -310,6 +310,110 @@ public void DoAbort()
 
             public abstract void InvokeCallback(bool allowPooling);
 
+            internal virtual bool TryBatch(SocketAsyncContext context, int id, ref Interop.Sys.IoControlBlock ioControlBlock) => false;
+
+            internal virtual void HandleBatchEvent(in Interop.Sys.IoEvent ioControlBlock)
```

Comment:
minor: using `in` to avoid expensive struct copies on Linux

Reply:
It only works for `readonly` structs AFAIK.